### ISSUE TITLE
Challenge-Findable

### ISSUE THREAD

**Issue by selgebali** [LibraryCarpentry/lc-fair-research#66]
The challenges below are taken from the Sprint [GoogleDoc](https://docs.google.com/document/d/124pu7Ufy3QKzGhowId2PtwvQ5LNJU35UpZ9qxbl1ZzM/edit?ts=5f479d7d#). 

**Challenge 1:**
arXiv is a preprint repository for physics, math, computer science and related disciplines. It allows researchers to share and access their work before it is formally published. 
Go to arXiv and 
Does arXiv use a DOI?
Compare these two papers:
https://arxiv.org/abs/2008.09350 
https://arxiv.org/abs/2008.00287
which one of them has a persistent identifier?

**Challenge 2:**
Look at this paper [link]. Click on the ‘pdf’ link to download it. Do a full-text search by using control + F or command + F and search for ‘http’. Did the author use DOIs for their data and software?

Challenge 3:
What is the problem with referring to your code and software only with a URL [example] without providing a DOI?  

**Is anyone interested in expanding on these exercises and reviewing them? You could also add them to the main lesson under Findable.  Don't forget to add the solution to the lesson.** 

Don't forget to check out the comments in the [google doc](https://docs.google.com/document/d/124pu7Ufy3QKzGhowId2PtwvQ5LNJU35UpZ9qxbl1ZzM/edit?ts=5f479d7d#). 

Any further comments on these challenges please add them to this issue!

**Comment by aromanowski** [comment]
Hi!
I am in the process of developing a 'data management' lesson, which includes FAIR guidelines. Some of the challenges we were considering is the following:

### (Findable) 

1) Find the image datasets from Wenden et al., PNAS 2012.  
Link: https://www.pnas.org/content/109/17/6757/tab-figures-data 
(there aren’t any links to the datasets, part of the data is shared as .wmv and .avi movies). 

2) Find the protocol for luminescence imaging.  
Wenden et al., PNAS 2012 -> Edwards et al, MSB 2010 -> Gould et al., TPC 2006 (3 hops). 
(not efficient, takes 3 hops to find the protocol -> leads to discussion, what could have been done better?)

3) Find the raw RNA-seq data in Li et al., Genes Dev. 2012. 
Link: http://genesdev.cshlp.org/content/26/8/785.long 
(raw data has not been deposited!) 

All the best,
Andrés

**Comment by libcce** [comment]
@aromanowski these are great examples. @selgebali wonder if one of these we can use for the lesson episode? @aromanowski is it fine if we use one of these examples? Thanks!

**Comment by ahvyoung** [comment]
I agree with the comment in the Google Doc that challenge 1 could be confusing when it comes to claryfiying the concept of PIDs. I wonder if it would be better to use an article that has a DOI and then an article that does not have a DOI?

**Comment by libcce** [comment]
@ahvyoung that might be a clearer example TBH. arXiv has been around so long it kind of predates the big push for DOIs for preprints. The information about arXiv might be best for a callout instead, to note some of the other examples out there?

**Comment by aromanowski** [comment]
Hi @libcce! Yes, of course you may use them. We will likely implement them on the course we are developing for biological scientists: https://github.com/carpentries-incubator/fair-bio-practice

**Comment by rbarthau** [comment]
Hello all,
maybe one idea for an exercise for challenge 3:
Provide a list with several links (a non-functioning URL, a tombstone page, a workind DOI and optionally a  version of a DOI or a fragmentation) and ask if the participants can detect the problems and advantages there. They might post their answers in a shared document.

- Link 1: 404-error - Page not found

Problem: a normal URL that is out of date, data or publication can no longer be cited correctly
Advantage: none

- Link 2: Tombstone page

Problem: Publication or data are no longer accessible
Advantage: contains information/full citation of item, might contain link to new version, might give reason for unavailability

- Link 3: working DOI

Problem: the question might come up, what happens if I update my data/publication
Advantage: permanently citable

- Link 4: Version of an existing DOI (or DOI with DOI fragments?)

Problem: versioning is not really wanted for PIDs
Advantage: it is visible that (versioning or fragmentation) is still possible

(sorry, I haven't looked up any links yet. If my idea is unclear or if the links are urgently needed, I will be happy to look for some.)

**Comment by aromanowski** [comment]
Just, if you use any of the proposed challenges, remember to acknowledge our lesson ;) https://github.com/carpentries-incubator/fair-bio-practice
